src/webob/acceptparse.py

```diff
@@ -1250,7 +1263,11 @@ def acceptable_offers(self, offers):
                  where each offer in `offers` is paired with the qvalue of 1.0,
                  in the same order as in `offers`.
         """
-        return [(offer, 1.0) for offer in offers]
+        return [
+            (offers[offer_index], 1.0)
+            for offer_index, _, _
+            in self._parse_and_normalize_offers(offers)
+        ]
```

Comment:
Could we add a comment here to explain why we are parsing even though it might seem unnecessary? Something like `We call _parse_and_normalize_offers() here so that invalid offers are dropped as in  AcceptValidHeader.acceptable_offers.`?

Reply:
fixed